modules/ocf_desktop/files/xsession/lightdm/lightdm-gtk-greeter.po

```diff
@@ -0,0 +1,3 @@
+#: ../src/lightdm-gtk-greeter.c:1479
```

Comment:
I see now. I found [this page](https://www.gnu.org/software/gettext/manual/html_node/PO-Files.html) which says

>  It is possible to have several entries with the same untranslated-string in a PO file, provided that they each have a different context.

but it's referring to a field called `msgctxt`, not the English word.